course6/toyapp_mikhail/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+project(toyapp)
+cmake_minimum_required(VERSION 3.0)
+
+# Ugly way of providing the path on Windows
+#include_directories(D:/SDL)
+#LINK_DIRECTORIES(D:/SDL/lib/win64)
```

Comment:
Was not actually reading / editing these files. That's what I copied from toyapp generic folder.
I plan to review and modify all CMakeList.txt in next pull request.

Will it work for Linux if I add these includes for windows? Currently I don't even need it because I installed SDL2 on system level. Is it OK?